tests/migration/online_migration/yast2_migration.pm

```diff
@@ -209,10 +209,15 @@ sub run {
     assert_screen 'yast2-migration-target-list-selected', 60;
     send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 20, 3;
     send_key "alt-n";
-    # in leap to sle migration, we need click trust for package-hub's import-untrusted-gpg-key
-    if (is_leap_migration) {
+    # migration via smt will install packagehub and NVIDIA compute, we need click trust
+    # gpg keys; Same with leap to sle migration, need to trust packagehub gpg key.
+    if ((get_var('SMT_URL') =~ /smt/) || (is_leap_migration)) {
```

Comment:
Why it is not needed for pscc online migration with phub?

Reply:
The untrusted key will be imported when packagehub is installed for the first time. For pscc cases, the key had been imported in zypper_patch module. For smt cases, when doing zypper_patch, the key is not imported because packagehub is not registered here, it is installed when doing yast2_migration for the first time.

Reply:
Ok, it makes sense.